submissions/nadsatt/html-css-popup/index.html

```diff
@@ -0,0 +1,294 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>popup</title>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        helllo
+    </head>
+    <body>
+        <nav class="nav">
+            <ul class="list">
+                <li class="list-item">
+                    <a class="list-item__link" href="#">Mail</a>
+                </li>
+                <li class="list-item">
+                    <a class="list-item__link" href="#">Images</a>
+                </li>
+                <li class="list-item">
+                    <input class="list-item__checkbox" type="checkbox">
+                    <a class="list-item__link">
+                        <svg class="link__icon" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="24" height="24" viewBox="0 0 512 512" enable-background="new 0 0 512 512" xml:space="preserve">
+                            <path d="M186.2,139.6h139.6V0H186.2V139.6z M372.4,0v139.6H512V0H372.4z M0,139.6h139.6V0H0V139.6z M186.2,325.8h139.6V186.2H186.2 V325.8z M372.4,325.8H512V186.2H372.4V325.8z M0,325.8h139.6V186.2H0V325.8z M186.2,512h139.6V372.4H186.2V512z M372.4,512H512 V372.4H372.4V512z M0,512h139.6V372.4H0V512z"/>
+                        </svg>
+                    </a>
+                    <div class="list-item__popup">
+                       <div class="popup__items--visible">
```

Comment:
If you use BEM methodology use class-modificator for the state of elements. Also, class-modificator can't be without this  class no modificator.
[Read more about modificator](https://ru.bem.info/methodology/quick-start/#%D0%9C%D0%BE%D0%B4%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%82%D0%BE%D1%80)
BEM element can't exist outside his BEM block. I can't see block `.popup` of element `.popup__items`.